### DISCUSSION TITLE
Redeploy on Env change?

### DISCUSSION THREAD

**Discussion by ohashioji** [vercel/community#445] [Help] [answered]
Is there a way to redeploy a production environment after updating env variables without pushing to git?

**Comment by Pieparker** [comment, marked as answer]
Sure – you can [redeploy](https://vercel.com/docs/concepts/deployments/redeploying)! This can be performed in the UI. You can use your browser's dev tools to mimic the `redeploy` API call which we make, and perform this straight via the API yourself too.

**Reply by ohashioji** [reply]
Awesome thanks for the answer!
